lasagne/nonlinearities.py

```diff
@@ -22,10 +82,31 @@ def rectify(x):
 
 
 # leaky rectify
-# Maas et al: Rectifier Nonlinearities Improve Neural Network Acoustic Models
-# http://web.stanford.edu/~awni/papers/relu_hybrid_icml2013_final.pdf
 class LeakyRectify(object):
+    """Implementation of a leaky rectifier
+    :math:`\\varphi(x) = \max(leakiness \cdot x, x)`.
+
+    Parameters
+    ----------
+    leakiness : float in [0, 1]
```

Comment:
Hmm, here the range is really useful to have as part of the type. Maybe just ignore my comment on the return type further above. Also `float` is probably okay for a type even if it can be a Theano scalar instead.